docs/source/tutorial_notebooks/getting_started_1_register_datasets.ipynb

```diff
@@ -420,7 +417,7 @@
    "id": "8f6909cf-9826-4533-8b91-bb7a63079b37",
    "metadata": {},
    "source": [
-    "## 6) Special cases\n",
+    "## 7) Special cases\n",
```

Comment:
I would modify the end of the first sentence under the heading "Registering external datasets" as follows:

> ..and to physically manage the data (either copy the cata to the central `root_dir` or verify that it already exists there).